bamboo/unit_tests/test_unit_check_proto_models.py

```diff
@@ -24,15 +25,41 @@ def skeleton_models(cluster, dir_name, executables, compiler_name):
                 data_filename_train_default=None
                 data_filedir_test_default=None
                 data_filename_test_default=None
+                data_reader_path=None
                 if 'mnist' in file_name:
                     data_filedir_default = '/p/lscratchf/brainusr/datasets/MNIST'
                     data_reader_name = 'mnist'
+                elif 'adversarial' in file_name:
+		    data_filedir_default = '/p/lscratchf/brainusr/datasets/MNIST'
+                    data_reader_path = '%s/model_zoo/models/gan/mnist/adversarial_data.prototext' % (dir_name)
```

Comment:
Indent is off on this line.

Reply:
strange I wonder why python wasn't complaining about that. good catch